### DISCUSSION TITLE
Any interest in whitespaces / end-of-file / mixed-line ending fixes?

### DISCUSSION THREAD

**Discussion by bocan** [adityatelange/hugo-PaperMod#1584] [Q&A]
Before I consider raising a PR, any interest in a bunch of whitespaces cleanups?  I added an optional pre-commit configuration file to the repo, and ran it via `pre-commit run --all-files`, and it found a number of things to fix:

https://github.com/adityatelange/hugo-PaperMod/compare/master...bocan:hugo-PaperMod:main 

AFAICT, this should have no impact on any functionality.

**Comment by adityatelange** [comment]
Yes why not but this will keep on adding. Any way to automate it?

**Reply by bocan** [reply]
With the `.pre-commit-config.yaml` file, locally, one can run `pre-commit install`, then every time you run a commit, git will check it, fix any broken files, then you can `git add` them, and repeat.

Or, to automate it fully with a github action, you could add something like [this](https://github.com/pre-commit/action).
